### PR TITLE
proofreading changes

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

**Comment by craigcomstock** [comment]
Am reading through this entire document just to keep it fresh in my mind and will add any bits I think need fixing. One thing I wonder about is spacing around '/'. I don't think we need spaces and should be consistent. Such as "cannot / should" versus "cannot/should". Oxford style guide doesn't mention this directly but in its explanatory text uses no spaces consistently.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -270,9 +270,9 @@ For other integer types without a format, cast `signed` types to `intmax_t` and
 Logging Conventions
 -----------------------------------
 
-CFEngine outputs messages about what its doing using the `Log()` function. It
-takes a `LogLevel` enum mapping closely to syslog priorities. Please try to do
-the following when writing output messages.
+CFEngine outputs messages about what it's doing using the `Log()` function. It
+takes a `LogLevel` enum mapping closely related to syslog priorities. Please try
```
Comment (olehermanse) [inline CONTRIBUTING.md]: ~related~

I prefer how it was before, but you can maybe add a comma after `enum`.

Comment (craigcomstock) [reply CONTRIBUTING.md]: The comma helps me. Otherwise I was reading it as "enum mapping" instead of an "enum (which is mapping) ..." :+1:

**Comment by craigcomstock** [comment]
started this as an edit in github.com but would rather have as a branch from my own account.

**Pull request closed without merging**